### ISSUE TITLE
Multiline RUN commands are not possible without spaces

### ISSUE THREAD

**Issue by GeertJohan** [vladaionescu/earthly#59]
First of all, thanks for this interesting new tool!

I have a Makefile that I'm converting to Earth and am running into a weird issue. Consider the following snipet:

```Makefile
.PHONY: _gengo
_gengo:
	protoc \
		-I=. \
		-I=/src \
		-I=/src/github.com/gogo/protobuf/protobuf \
		--gogofaster_out=\
plugins=grpc,\
Mgoogle/protobuf/any.proto=github.com/gogo/protobuf/types,\
Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types,\
Mgoogle/protobuf/struct.proto=github.com/gogo/protobuf/types,\
Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types,\
Mgoogle/protobuf/wrappers.proto=github.com/gogo/protobuf/types,\
Mgoogle/protobuf/empty.proto=github.com/gogo/protobuf/types:\
. \
		./dex.proto
```

It is important to protoc that the value for `--gogofaster_out` is a continuous string without spaces. But I'm having trouble getting Earthly to accept the `\` escape if it's not preceded by a space.
So when I use the following in my `earth.build`:

```go
generate-go:
	FROM golang:1-stretch
	RUN go get github.com/gogo/protobuf/protoc-gen-gogofaster
	COPY +protoc/out/bin/protoc /usr/local/bin/protoc
	COPY +gogo-protobuf/out/src /gogo-proto-src
	COPY dex.proto .
	RUN protoc \
		-I=. \
		-I=/gogo-proto-src \
		-I=/gogo-proto-src/github.com/gogo/protobuf/protobuf \
		--gogofaster_out=\
plugins=grpc,\
Mgoogle/protobuf/any.proto=github.com/gogo/protobuf/types,\
Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types,\
Mgoogle/protobuf/struct.proto=github.com/gogo/protobuf/types,\
Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types,\
Mgoogle/protobuf/wrappers.proto=github.com/gogo/protobuf/types,\
Mgoogle/protobuf/empty.proto=github.com/gogo/protobuf/types:\
. \
		dex.proto
```

I get these errors:
```
line 41:0 token recognition error at: 'plugins='
line 41:8 token recognition error at: 'grpc,'
line 43:1 token recognition error at: 'google/'
line 43:8 token recognition error at: 'protobuf/'
line 43:17 token recognition error at: 'duration.proto='
line 43:32 token recognition error at: 'github.com/'
line 43:43 token recognition error at: 'gogo/'
line 43:48 token recognition error at: 'protobuf/'
line 43:57 token recognition error at: 'types,'
```

When I add a space before the `\`'s, earth can parse the file and starts properly, but protoc doesn't accept the arguments as they are now separate arguments instead of a single value to `--gogofaster_out`.

I can remove the newlines and have a single long line, but that's not very readable/maintanable. I don't understand why `\` need a space before it to escape the newline.

**Comment by vladaionescu** [comment]
Hi @GeertJohan - thanks for the detailed report! It's a [known bug](https://github.com/vladaionescu/earthly/issues/17).

Your use-case seems pretty compelling to increase the priority of this. I'll look into this.

**Comment by vladaionescu** [comment]
Hi @GeertJohan - this is now fixed in the new release `v0.1.1`. Simple re-run the [install command](https://github.com/vladaionescu/earthly#installation) to replace your current version.

Any issues, let me know!

**Comment by GeertJohan** [comment]
It works as expected now! Thanks :tada:
